src/Components/InputWithLabel.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from "prop-types";
+
+const InputWithLabel = ({todoTitle, handleTitleChange, children}) => {
+  const inputRef = React.useRef();
+  React.useEffect(() => {
+      inputRef.current.focus(); 
+  }, [])
+  return (
+    <>
+    <label htmlFor="todoTitle">{children}</label>
+    <input 
+      id="todoTitle"
+      type="text" 
+      name="title" 
+      value={todoTitle}
+      onChange={handleTitleChange}
+      ref={inputRef}
+    />
+    </>
+  )
+}
+
+InputWithLabel.propTypes = {
```

Comment:
todoTitle, handleTitleChange, children are the only props being defined for InputWithLabel (see line 4 in this file)